client/state/data-layer/wpcom/sites/comments/likes/mine/delete/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+	COMMENTS_LIKE,
+	COMMENTS_UNLIKE,
+} from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { local } from 'state/data-layer/utils';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+
+export const unlikeComment = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'POST',
+		apiVersion: '1.1',
+		path: `/sites/${ action.siteId }/comments/${ action.commentId }/likes/mine/delete`
+	}, action ) );
+};
+
+export const updateCommentLikes = ( { dispatch }, { siteId, postId, commentId }, next, { i_like, like_count } ) => dispatch( local( {
+	type: COMMENTS_LIKE,
+	siteId,
+	postId,
```

Comment:
does the post id get used? is it even relevant? I know this is a bit murky since WP ids comments by site and comment id while still preserving a link to the original post. it didn't seem like that was being used here.

maybe it's worth passing even if it's not used…